### ISSUE TITLE
Disparity

### ISSUE THREAD

**Issue by jpyusiong** [mrharicot/monodepth#96]
Hi,
Is this formula for the disparity only applicable to the KITTI dataset or Cityscapes dataset?
disp = 0.3 * self.conv(x, 2, 3, 1, tf.nn.sigmoid)

Is it possible to use disp = self.conv(x, 2, 3, 1, tf.nn.sigmoid)? I am just wondering if the constant 0.3 significantly alters the disparity values.

Thank you.

**Comment by mrharicot** [comment]
Hi,
The 0.3 constant originally came just to put a maximum on the disparity.
0.3 is actually quite large, the maximum disparity in KITTI is about 0.15 or so.
If you are using a different dataset with different characteristics, you might want to change this value.

**Comment by jpyusiong** [comment]
Thanks for the insights.
